### DISCUSSION TITLE
Question：Possible to assign certain apps to use proxy

### DISCUSSION THREAD

**Discussion by mlflabs** [shadowsocks/shadowsocks-windows#3192] [Q&A] [answered]
Hello, 
Have been using this client for a while, and it's really good, and stable. thx.

Just wondering about PAC, is it possible to assign some apps to use the proxy and some not to.

For example, if I have Wechat installed, I don't want to use the proxy, for others I do

Mike

**Comment by chenshaoju** [comment, marked as answer]
There is a different solution: You need to create a new windows account, And running WeChat as the new windows account.

The PAC only works with the current users.

It's a bit complex:
http://woshub.com/run-program-as-different-user-windows/

**Reply by mlflabs** [reply]
The article is really good, thx a lot
